README.md

```diff
@@ -4,13 +4,54 @@ The CHAMMI dataset is available for download on [Zenado](https://zenodo.org/reco
 
 We released model checkpoints and the training scripts to further aid reproducibility. 
 Our code and checkpoints are now publicly available at [here](https://github.com/chaudatascience/channel_adaptive_models).
+
+## Disclosure
+
+These instructions are made for conda environments, but the `requirements.txt` file is given for other users
+
+### Notice
```

Comment:
Move this title up to include the previous statement.
Change the title from Notice to Note (keep the three hashes)